hmwk/plate_survey_overlap.ply

```diff
@@ -0,0 +1,25 @@
+4 polygons
```

Comment:
In your polygon files, you clearly had the initiative and understanding to realize that you had to construct polygons consisting of the intersection of the rectangular "imaging area" with the circular caps "spectroscopic plates". So, I can certainly tell that you knew what you were doing.

However, because two of the caps are entirely enclosed by the rectangle (`polygon 2` and `polygon 3` in your file) you don't need to intersect those with the rectangle. For those polygons, you could just have used the representation for the single cap. 

Further, you only technically needed to include the cap that represents the left-hand (or right-hand) edge of the rectangle for `polygon 1` and `polygon 4`, i.e. you just needed to include the side of the rectangle that clipped a particular cap.

In conclusion, this file _could_ have been reduced to:
```
4 polygons
polygon 1 (2 caps, 1 weight, 0 pixel, 0 str):
-0.7513632077525959 0.35036641783094075 0.5591929034707469 0.0006091729809042379 
-0.44228869021900136 -0.8968727415326883 0.0 1.0 
polygon 2 (1 caps, 1 weight, 0 pixel, 0 str):
-0.7552824306520477 0.2899257614214516 0.5877852522924731 0.0006091729809042379 
polygon 3 (1 caps, 1 weight, 0 pixel, 0 str):
-0.7928125735064364 0.24238712836711696 0.5591929034707469 0.0006091729809042379 
polygon 4 (2 caps, 1 weight, 0 pixel, 0 str):
-0.7882819412214738 0.1819892258667493 0.5877852522924731 0.0006091729809042379 
-0.19509032201612866 -0.9807852804032303 0.0 -1.0 
```

I appreciate that you were trying to automate the process as much as possible, so I only dinged you a very tiny amount in the _compactness/optimal_ category.